core/data_category.js

```diff
@@ -98,6 +98,8 @@ Blockly.DataCategory.addDataVariable = function(xmlList, variable) {
   //    <field name="VARIABLE">variablename</field>
   // </block>
   Blockly.DataCategory.addBlock(xmlList, variable, 'data_variable', 'VARIABLE');
+  // In the flyout, this ID must match variable ID for monitor syncing reasons
```

Comment:
LG but add an id to the xml on line 97 (`id="variableId"` or something similar)

Reply:
👍  4dc0e84